server/conf/messages.en-US

```diff
@@ -59,7 +59,7 @@ content.mobileFileUploadHelp=On your phone? “Choose File” also allows you to
 content.noValidAddress=We could not verify the address that you provided. Please edit your address to continue.
 content.notEligible=Not Eligible
 content.or=or
-content.pleaseCreateAccount=If you sign into a city account now, your information will be saved and you''ll be able to return at any time to complete future applications.  If you do not already have an account, the login page will allow you to register.  You won''t lose any data you''ve already entered.
+content.pleaseCreateAccount=Log in to your {0} account or create an account to have your information saved. You''ll be able to return at any time to complete future applications. If you do not already have an account, the login page will allow you to sign up and you won''t lose any data you''ve already entered.
```

Comment:
```suggestion
content.pleaseCreateAccount=Log in to your {0} account or create an account to have your information saved. You'll be able to return at any time to complete future applications. If you do not already have an account, the login page will allow you to sign up and you won't lose any data you've already entered.
```

Reply:
We need the double apostrophes:
https://github.com/civiform/civiform/blob/8b443012c970ab8c46816548906f81b39869155b/server/conf/messages#L5